pandas/tests/series/test_alter_axes.py

```diff
@@ -55,6 +55,13 @@ def test_rename(self):
         renamed = renamer.rename({})
         self.assertEqual(renamed.index.name, renamer.index.name)
 
+    def test_rename_by_series(self):
+        s = Series(range(5), name='foo')
+        renamer = Series({1: 10, 2: 20})
```

Comment:
These should all be in `test_generic` now. Initially I didn't understand how `test_generic` was setup, so I added the test here. I *think* these tests are all redundant, and could remove them this one and `test_rename` above, but I always worry about deleting tests :)